examples/gke/my_test.sh

```diff
@@ -36,8 +37,6 @@ main() {
     docker exec "$config_container_id" gcloud container clusters get-credentials my-cluster --project my-project --zone us-west1-a
     docker exec "$config_container_id" kubectl cluster-info
     docker exec "$config_container_id" chart_test.sh --config /workdir/.testenv
```

Comment:
Should `chart_test.sh` be updated to `ct` commands?

Reply:
Fixed.